tutorials/applications/QuantumEvolutionKernel_Tutorial.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
```

Comment:
<p>This last image is not rendering on the docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>That's strange, it's showing up when generating the doc locally...</p>